docs/source/intro.rst

```diff
@@ -0,0 +1,72 @@
+- a **large number of outliers** that can originate from various sources (e.g., photogrammetric blunders, clouds).
+
+These factors lead to difficulties in assessing the accuracy and precision of DEMs, which are necessary to perform further analysis.
+
+In ``xdem``, we provide a framework with state-of-the-art methods published in the scientific literature to make DEM calculations consistent, reproducible, and easy.
+
+Accuracy and precision
+**********************
+
+Both accuracy and precision are important factors to account for when analyzing DEMs:
+
+- the **accuracy** (systematic error) of a DEM describes how close a DEM is to the true location of measured elevations on the Earth's surface,
+- the **precision** (random error) of a DEM describes the typical spread of its error in measurement, independently of a possible bias from the true positioning.
+
+TODO: Add a little schematic!
+
+Absolute or relative accuracy
+*****************************
+
+The measure of accuracy can be further divided into two aspects:
+
+- the **absolute accuracy** of a DEM describes the average shift to the true positioning. Studies interested in analyzing features of a single DEM in relation to other georeferenced data might give great importance to this potential bias.
+- the **relative accuracy** of a DEM is the potential shifts, tilts, and deformations in relation to other elevation data that does not necessarily matches a given referencing. Studies interested in comparing DEMs between themselves might be only interested in this accuracy.
+
+TODO: Add another little schematic!
+
+Optimizing DEM absolute accuracy
+**********************************
+
+Shifts due to poor absolute accuracy are common in elevation datasets, and can be easily corrected by performing a DEM co-registration to precise and accurate, quality-controlled elevation data such as ICESat and ICESat-2.
+Quality-controlled DEMs aligned on high-accuracy data also exists, such as TanDEM-X global DEM (see `Rizzoli et al. (2017) <https://doi.org/10.1016/j.isprsjprs.2017.08.008>`_)
+
+Those biases can be corrected using the methods described in :ref:`coregistration`.
+
+TODO: Add a point data - DEM co-registration plot
+
+Optimizing DEM relative accuracy
+**********************************
+
+As the **absolute accuracy** can be corrected a posteriori using reference elevation datasets, many analyses only focus on **relative accuracy**, i.e. the remaining biases between several DEMs co-registered relative one to another.
+By harnessing the denser, nearly continuous sampling of raster DEMs (in opposition to the sparser sampling of higher-accuracy point elevation data), one can identify and correct other types of biases:
+
+- Terrain-related biases that can originate from the difference of resolution of DEMs, or instrument processing deformations.
```

Comment:
You could add some scientific references here, like you do for the variograms. For example the Gardelle paper here, and Luc's paper for the line below.